chats/apps/api/v1/rooms/serializers.py

```diff
@@ -67,7 +67,10 @@ class TransferRoomSerializer(serializers.ModelSerializer):
         allow_null=True,
     )
     queue_uuid = serializers.PrimaryKeyRelatedField(
-        queryset=Queue.objects.all(), required=False, source="queue", write_only=True
+        queryset=Queue.objects.exclude(is_deleted=True),
```

Comment:
Don't need to exclude here, it'll be possible to return the rooms from deleted queues. Better to add a new filter(is_deleted) on the viewset with the default excluding the deleted

Reply:
resolved.